app/helpers/govuk_link_helper.rb

```diff
@@ -43,7 +43,16 @@ def govuk_mail_to(email_address, name = nil, extra_options = {}, &block)
 
   def govuk_button_to(name = nil, options = nil, extra_options = {}, &block)
     extra_options = options if block_given?
-    html_options = build_html_options(extra_options, style: :button)
+    html_options = {
+      data: {module: "govuk-button"}
```

Comment:
```suggestion
      data: { module: "govuk-button" }
```

Rubocop should have caught this but it runs via `make ruby-lint` and predates this library's use of the `app` directory. I've updated that in #450